commands/scan/buildscan.go

```diff
@@ -186,7 +191,21 @@ func (bsc *BuildScanCommand) CommandName() string {
 	return "xr_build_scan"
 }
 
-func trimBuildScanResultUrl(fullUrl string) (string, string, error) {
+// There are two cases. when serverDetails.Url is configured and when serverDetails.XrayUrl and serverDetails.ArtifactoryUrl are configured
+// The function will retutn the Url if configured and will trim xray if serverDetails.Url is not configured
+func (bsc *BuildScanCommand) configuredCliUrl() (string, error) {
```

Comment:
you can create a different method. `func getActualUrl(config.ServerDetails) (string, error)` and than add test for this